packages/accounts-passwordless/passwordless_client.js

```diff
@@ -120,3 +121,6 @@ Accounts.autoLoginWithToken = function() {
     }
   });
 };
+
+// Run check for login token on page load
+document.addEventListener('DOMContentLoaded', () => Accounts.autoLoginWithToken())
```

Comment:
Auto login wasn't running by default so I had to add this so that when clicking on a link will login users.